### ISSUE TITLE
man pages not available when installed with pipx

### ISSUE THREAD

**Issue by jmdevin** [bpython/bpython#1023]
# Description
When installed via `pipx install bpython`, the man pages for bpython are not available (even the .rst files don't seem to be present).

I took a look at pipx's documentation and they linked to ([pycowsay](https://github.com/cs01/pycowsay/tree/master)) as an example. From what I can tell the only difference is pycowsay has the man pages built and included in the repo, whereas bpython just has the .rst files.

# Version info
OS: Ubuntu 24.04.01
pipx: 1.4.3
python: 3.12.3
bpython: 0.24

**Comment by sebastinas** [comment]
The man pages are only built and installed if sphinx is available.

**Comment by jmdevin** [comment]
Would it be possible to include the built pages in the repository? It would be useful for users with a situation like mine.

**Comment by jmdevin** [comment]
@sebastinas I tried installing sphinx with pipx, then installing bpython, but it still didn't build the man pages. I assume this is caused by pipx's manner of installing packages, so in this case I think it's more accurate to say that the man pages just aren't built if someone is using pipx.

**Comment by sebastinas** [comment]
So is pipx ignoring `data_files`?

**Comment by sebastinas** [comment]
No, it doesn't. But installing sphinx with pipx get sphinx installed in a different virtual env. So it's not available when building bpython.

**Comment by jmdevin** [comment]
Yes, I didn't quite have time to check the docs yesterday but I had started to suspect that. I don't have a definitive answer for whether pipx would generate the docs if sphinx were installed globally via regular pip or apt or something, but that isn't recommended anyway so it's somewhat irrelevant.

Overall, I think if you want to start tracking the generated man file as part of the repo itself that would be great, but otherwise we can close this issue as a "won't fix" since the actual problem has been identified.
